<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabLogistica</class>
 <widget class="QWidget" name="TabLogistica">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTabWidget" name="tabWidgetLogistica">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabAgendarColeta">
       <attribute name="title">
        <string>Agendar Coleta</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="WidgetLogisticaAgendarColeta" name="widgetAgendarColeta" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabColeta">
       <attribute name="title">
        <string>Coleta</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="WidgetLogisticaColeta" name="widgetColeta" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabRecebimento">
       <attribute name="title">
        <string>Recebimento</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_28">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="WidgetLogisticaRecebimento" name="widgetRecebimento" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabAgendarEntrega">
       <attribute name="title">
        <string>Agendar Entrega</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="WidgetLogisticaAgendarEntrega" name="widgetAgendaEntrega" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabEntregas">
       <attribute name="title">
        <string>Entregas</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_27">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="WidgetLogisticaEntregas" name="widgetCalendarioEntrega" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabCaminhoes">
       <attribute name="title">
        <string>Caminhões</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="WidgetLogisticaCaminhao" name="widgetCaminhao" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabRepresentacao">
       <attribute name="title">
        <string>Representação</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="WidgetLogisticaRepresentacao" name="widgetRepresentacao" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabEntregues">
       <attribute name="title">
        <string>Entregues</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="WidgetLogisticaEntregues" name="widgetEntregues" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabCalendario">
       <attribute name="title">
        <string>Calendário</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="WidgetLogisticaCalendario" name="widgetCalendario" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Devolução</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="WidgetDevolucao" name="widgetDevolucao" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QFrame" name="frameForn">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelCompras_11">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Fornecedores</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="TableView" name="tableForn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>src/tableview.h</header>
  </customwidget>
  <customwidget>
   <class>WidgetLogisticaRecebimento</class>
   <extends>QWidget</extends>
   <header>src/widgetlogisticarecebimento.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetLogisticaAgendarEntrega</class>
   <extends>QWidget</extends>
   <header>src/widgetlogisticaagendarentrega.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetLogisticaRepresentacao</class>
   <extends>QWidget</extends>
   <header>src/widgetlogisticarepresentacao.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetLogisticaAgendarColeta</class>
   <extends>QWidget</extends>
   <header>src/widgetlogisticaagendarcoleta.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetLogisticaEntregas</class>
   <extends>QWidget</extends>
   <header>src/widgetlogisticaentregas.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetLogisticaColeta</class>
   <extends>QWidget</extends>
   <header>src/widgetlogisticacoleta.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetLogisticaCaminhao</class>
   <extends>QWidget</extends>
   <header>src/widgetlogisticacaminhao.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetLogisticaEntregues</class>
   <extends>QWidget</extends>
   <header>src/widgetlogisticaentregues.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetLogisticaCalendario</class>
   <extends>QWidget</extends>
   <header>src/widgetlogisticacalendario.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetDevolucao</class>
   <extends>QWidget</extends>
   <header>src/widgetdevolucao.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
